### PR TITLE
Feat : "added My Cart Item"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added My Cart Item 
Mycart page

### CODE REVIEW

File: Pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <div class="badge-div">
+          <i class="fa fa-heart"></i>
+          <span class="badge-number">4</span>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <h1 class="text-center">MyCart</h1>
+      <div class="mycart-container">
+        <section class="cart-item">
```
Comment (ttahm3d) [inline Pages/cart.html]: Just a suggestion, enclose a `div` inside `section` not the other way around

**Review by ttahm3d — APPROVED** [review]
Have added one suggestion. Rest all looks good.

File: CSS/cart.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  padding: 1.5rem 1rem;
+  background-color: var(--off-white);
+  box-shadow: var(--box-shadow-light-card);
+}
+.cart-item-btn {
+  margin: 0.5rem 0;
+}
+.card-item-img {
+  height: max-content;
+  width: 250px;
```
Comment (vivekmehto) [inline CSS/cart.css]: use of fixed width is not recommended.

**Review by vivekmehto — COMMENTED** [review]
everything looks good just one suggestion don't use fixed-width it will create issues for different screen sizes.

File: CSS/cart.css (hunk 2 above)
Comment (sadathshariff) [reply CSS/cart.css]: removed it

**Pull request merged**